runs/set_module_version.go

```diff
@@ -0,0 +1,17 @@
+package runs
+
+import (
+	"fmt"
+	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+)
+
+func SetModuleVersion(cfg api.Config, workspace types.Workspace, version string) (*types.WorkspaceChangeset, error) {
```

Comment:
The usage of this function seems to use module@version as the version input. Does the API expect only the version?

Reply:
I built the API endpoint thinking that we would only allow the user to change the module version (thus the name). I ended up relying on the runConfigResolver to perform this override. I went back in and realized that the runConfigResolver actually allows both the module and the version to be overridden. Thus I am passing this in with the full module@version. I'm trying to move quick and not overthink. Should I rename the endpoint or change the runConfigResolver to just accept a version?

Reply:
👍 Awesome, I just wanted to confirm there wasn't a miscommunication between the user and API.
I think renaming the endpoint would be ideal.